projects/plugins/crm/includes/ZeroBSCRM.ScreenOptions.php

```diff
@@ -213,7 +213,12 @@ function zeroBS_outputScreenOptions(){
 
 	$screenOpts = $zbs->global_screen_options(); // phpcs:ignore WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase
 
-	?><script type="text/javascript">var zbsPageKey = '<?php echo esc_html( $zbs->pageKey ); ?>';var zbsScreenOptions = <?php echo json_encode($screenOpts); ?>;</script><?php
+	?>
+	<script type="text/javascript">
+	var zbsPageKey = '<?php echo esc_html( $zbs->pageKey ); /* phpcs:ignore WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase */ ?>';
```

Comment:
```suggestion
	var zbsPageKey = <?php echo wp_json_encode( $zbs->pageKey, JSON_UNESCAPED_SLASHES | JSON_HEX_TAG | JSON_HEX_AMP ); /* phpcs:ignore WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase */ ?>;
```
or, if the `esc_html()` really is needed rather than just being incorrect escaping for the context,
```suggestion
	var zbsPageKey = <?php echo wp_json_encode( esc_html( $zbs->pageKey ), JSON_UNESCAPED_SLASHES | JSON_HEX_TAG | JSON_HEX_AMP ); /* phpcs:ignore WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase */ ?>;
```